src/WebdriverClassicDriver.php

```diff
@@ -340,12 +347,16 @@ public function getAttribute(
         return $this->executeJsOnXpath($xpath, $script);
     }
 
+    /**
+     * {@inheritdoc}
+     * @return TElementValue
+     */
     public function getValue(
         #[Language('XPath')]
         string $xpath
     ) {
         $element = $this->findElement($xpath);
-        $widgetType = strtolower($element->getTagName() ?? '');
+        $widgetType = $element->getTagName();
```

Comment:
I found out that `getTagName` is guaranteed to return a lowercase string, so the code here is quite redundant (and removing it might additionally fix a bug with multibyte tag names).